host-os-build.py

```diff
@@ -29,6 +29,11 @@
 
 def main(args):
     CONF = utils.setup_default_config()
+    http_proxy = CONF.get('http_proxy')
+
+    if http_proxy:
+        utils.set_http_proxy_env(http_proxy)
+
```

Comment:
Shouldn't the proxy be usable for other scripts too? If so, place this inside setup_default_config().